include/bsmHandler.hpp

```diff
@@ -124,6 +124,9 @@ class BSMHandler {
         static constexpr uint32_t kIdRedactFlag       = 0x1 << 2;
         static constexpr uint32_t kSizeRedactFlag     = 0x1 << 4;
         static constexpr uint32_t kGeneralRedactFlag  = 0x1 << 8;
+        static constexpr int J2735_SPEED_UNAVAILABLE = 8191;
```

Comment:
question: where are these values coming from? can we add a comment letting devs know this?

Reply:
J2735 and the constant's name is fairly indicative of that to be honest. I can add a comment I suppose.